pims/cine.py

```diff
@@ -520,7 +522,12 @@ def __len__(self):
 
     def get_time(self, i):
         '''Returm the time of frame i in seconds.'''
-        return float(i) / self.frame_rate
+        # TODO: This is not guaranteed to be the actual time.
```

Comment:
At high frame rates this is probably true. At low frame rates the cameras really didn't seem to care.